integration-tests/src/test/java/com/datastax/oss/driver/core/cql/PreparedStatementIT.java

```diff
@@ -444,12 +445,38 @@ public void should_fail_fast_if_id_changes_on_reprepare() {
       executeDdl("DROP TABLE prepared_statement_test");
       executeDdl("CREATE TABLE prepared_statement_test (a int PRIMARY KEY, b int, c int)");
 
-      assertThatThrownBy(() -> session.execute(preparedStatement.bind(1)))
-          .isInstanceOf(IllegalStateException.class)
-          .hasMessageContaining("ID mismatch while trying to reprepare");
+      return assertThatCode(() -> session.execute(preparedStatement.bind(1)));
     }
   }
 
+  @BackendRequirement(type = BackendType.DSE)
```

Comment:
DSE is actually a different code base; it has to explicitly bring changes from the OSS C* project along.  I'm wondering if DSE has even brought the relevant change over to its code base (let alone released it yet).

It doesn't _look like_ that's happened yet (although I haven't made an exhaustive search).  Perhaps add a comment here saying that we'll need to add version boundaries to this guy as well once DSE brings the change over.

Reply:
Yes it's currently `4.0.0`: https://github.com/riptano/bdp/blob/6.8-dev/build.properties#L31, updated the description and added a comment